sources/modules/compilation-model/buildsystems.md

```diff
@@ -0,0 +1,115 @@
+## Module name: Build systems
+
+_Skeleton descriptions are typeset in italic text,_
+_so please don't remove these descriptions when editing the topic._
+
+### Overview
+
+_Provides a short natural language abstract of the module’s contents._
+_Specifies the different levels of teaching._
+
+------------------------------------------------------------------------
+Level             Objective
+----------------- ------------------------------------------------------
+Foundational      --- Knowledge about build systems
+
+Main              --- Usage of build system to compile a executable
+
+Advanced          --- Usage of build system to compile a library or
+                      add external libraries as a dependencies  
+
```

Comment:
Here it is said that "add external libraries as dependencies" is advanced, but it below it is in main.

Reply:
Here, we mean to add some external library like VTK or OpenCV and not compiling your own library.

Reply:
I fixed it with the latest commit.